### ISSUE TITLE
Build - Broken Aria-Reference will not be announced by screen readers

### ISSUE THREAD

**Issue by hydegeoff** [mmiron-HC/WST-RSAMS#152]
URL: https://rsams.dev.cloud.openplus.ca/en/user/
User Agent: Chrome v99 + WAVE plug-in v3.16
Language: EN and FR pages

Issue Description:
An aria-labelledby or aria-describedby reference exists, but the target for the reference does not exist.  This means broken aria reference attributes means that screen readers will not announce it when it is expected.
![Dashboard - Broken Aria reference](https://user-images.githubusercontent.com/88147013/161784548-a44e4b44-1f10-4ef8-8507-2e6c64bf1d09.jpg)

Steps to reproduce:

1. Login into RSAMs
2. When on the dashboard, select Build on one of the recall or alerts row items   
3. On the Build page, turn on WAVE plugin
Problem: there is a broken aria under the Workflow section on the right side of the page.

Expected results:
All aria references

**Comment by davidgotlieb** [comment]
No longer happening, so seems like it has been fixed up stream.
